src/backend/base/langflow/components/vectorstores/AstraDB.py

```diff
@@ -59,6 +93,19 @@ class AstraVectorStoreComponent(LCVectorStoreComponent):
             info="Optional namespace within Astra DB to use for the collection.",
             advanced=True,
         ),
+        DropdownInput(
```

Comment:
should we default to one or the other?

Reply:
Good point! For compatibility, i suppose defaulting to Embeddings makes sense so we dont break existing flows that were using it. (we are currently breaking Vectorize-based flows as mentioned...)

Side note, you may be wondering why its a dropdown rather than a boolean input / toggle... i tried to use the switch, but for whatever reason when disabling the flag, it didnt trigger the call to update_build_config - something i want to bring up with people on the langflow team...